groups/groups.yaml

```diff
@@ -318,23 +319,32 @@ groups:
     settings:
       ReconcileMembers: "true"
     members:
+      - k8s-infra-release-admins@kubernetes.io
```

Comment:
This change is not directly related to CRI tools. Just doing it as a cleanup since I was already in the file.